samples/config.default.js

```diff
@@ -1,36 +1,57 @@
-// User-specific configuration
-exports.voice = 'en-US_MichaelVoice';
-exports.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
+// User-specific CONFIGURATION
+exports.config = {}
+exports.config.voice = {"gender": "female", "language": "en-US"};
```

Comment:
what are some other examples of params you can add here?  or is it just `gender` and `language`?